pkg/cluster/cache_whitebox_test.go

```diff
@@ -107,13 +107,13 @@ func TestGetClustersByType(t *testing.T) {
 			// empty cache
 
 			//when
-			clusters := clusterCache.getCachedToolchainClustersByType(Member)
+			clusters := clusterCache.getCachedToolchainClusters()
```

Comment:
i think yes, will remove this test

Reply:
As far as I understand the logic behind the `ToolchainCluster` objects, there never is a place and time where a `ToolchainCluster` for host would co-exist with `ToolchainCluster` for members.

I.e. host-operator only looks at its namespace for member cluster objects and member operator(s) look at their namespaces for a single host cluster object in each.

The cache is actually keyed merely by the cluster name (which makes sense given the usage restrictions above) so there really isn't any assumption that the `ToolchainCluster` objects would be sourced from multiple namespaces IMHO (because then the name is theoretically not enough to distinguish them).

Therefore, IMHO, this test doesn't make sense anymore. In the past, there may have been different assumptions about the `ToolchainCluster` objects but since we're dropping the type, we have to work with the assumptions above.

Reply:
okay, i think i got the point now, i shall drop the test "`get clusters by type`" and keep `get host cluster` and `get member clusters` test